src/Sentry.Maui/Internal/MauiEventsBinder.cs

```diff
@@ -45,8 +54,6 @@ public void HandleApplicationEvents(Application application, bool bind = true)
             }
 
             // Navigation events
-            application.PageAppearing += OnApplicationOnPageAppearing;
```

Comment:
We're already subscribing to each `Page` methods below so this creates duplicate breadcrumbs for the same kind of event.

Should we remove the other one instead so we can reduce the number of subscribed events?